src/common/constants/Entities.ts

```diff
@@ -5,5 +5,6 @@ export const ENTITIES = {
   TEMPERATURE_BREACH_CONFIGURATION: 'TemperatureBreachConfiguration',
   SENSOR_LOG: 'SensorLog',
   SETTING: 'Setting',
-  SYNC_LOG: 'SyncLog'
+  SYNC_LOG: 'SyncLog',
+  SYNC_CONFIG: 'SyncConfig',
```

Comment:
Went with a new table mostly because it was quick. Open to using existing settings table if we think that's a better way to go!

Reply:
Resolving as issue created (#63).